hubblestack/extmods/grains/cloud_details.py

```diff
@@ -44,8 +43,6 @@ def _get_aws_details():
         # Else it will throw an Exception
         aws['cloud_account_id'] = int(aws['cloud_account_id'])
 
-        aws['aws_ami_id'] = requests.get('http://169.254.169.254/latest/meta-data/ami-id',
-                                         timeout=1).text
```

Comment:
Do we want to remove this? I'm pretty sure DME finds this useful since they have a standard parent AMI.

Reply:
I was talking it over with Scott, and with the instance id there, it's trivial to tie it back to the ami id with mavlink. There's not really a reason to have it in every single hubble event.

Reply:
@sumanmehta is this change acceptable, given that you can tie the instance ID to the ami ID pretty easily later? Reducing the number of fields we add to splunk events has a huge effect over millions of events.

Reply:
osquery allows easily to get additional aws instance metadata with a query if needed (https://osquery.io/schema/2.11.2#ec2_instance_metadata). This does not work for Azure, though. Throwing ami_id data in with _every_ event just adds to unnecessary storage pressure.

Reply:
@fossam Do you also plan to remove account-id as it also seems redundant given that instance ids are unique for AWS? As of now, we do not use this info, rather we use config data provided by Mavlink as it gives information about all the instances even where Hubble is not running.  So I am fine if we remove this field.

Reply:
@sumanmehta For now, I don't plan to remove account id because we've had times when we had account id from Hubble and nothing from anywhere else. If you have such situation attributing the instance to somebody becomes challenging without account id. 
It would be fine to have it reporting once an hour in a different sourcetype, but as it is I don't think it is.